ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,69 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    const vismetMetrics = [
+      'ContentfulSpeedIndex',
+      'FirstVisualChange',
+      'LastVisualChange',
+      'PerceptualSpeedIndex',
+      'SpeedIndex',
+      'videoRecordingStart',
+    ];
+    // eslint-disable-next-line array-callback-return
+    const contains = vismetMetrics.filter((metric) => title.includes(metric));
+    console.log('contains');
+    console.log(contains);
+    return contains.length > 0;
+  };
+
+  anyNotVismet = (alertSummary) => {
```

Comment:
this is to help avoid the visual metrics alert items because they have a different job from the pageload, therefore there's no browsertime-results.tgz for them.

Reply:
This is a technical detail. Why not give it a clearer name like `anyTestsWithBeforeAfterArtifacts()`?